test/framework/modules.py

```diff
@@ -404,6 +404,10 @@ def test_modpath_extensions_for(self):
         error_pattern = "Can't get value from a non-existing module"
         self.assertErrorRegex(EasyBuildError, error_pattern, self.modtool.modpath_extensions_for, ['nosuchmodule/1.2'])
 
+        realhome = os.environ.get('HOME')
+        os.environ['HOME'] = os.path.join(self.test_prefix, 'HOME')
+        os.makedirs("%s/modules/Compiler/GCC/4.7.2" % os.environ['HOME'])
```

Comment:
Because with "module show" the if statement in the module file is parsed, so for
```
 if isDir("/home/foo/modules/Compiler/GCC/4.7.2") then
         prepend_path("MODULEPATH", "/home/foo/modules/Compiler/GCC/4.7.2")
 end
```
`module show` will output just:
```
prepend_path("MODULEPATH", "/home/foo/modules/Compiler/GCC/4.7.2")
```
only if that directory exists, and nothing if it does not exist.